templates/zerver/emails/find_team.source.html

```diff
@@ -5,19 +5,13 @@
 {% endblock %}
 
 {% block content %}
-<p>{% trans %}Hi {{ user_name }},{% endtrans %}</p>
+<p>{% trans %}Thanks for your request! Your email address {{ email }} is associated with the following Zulip accounts:{% endtrans %}</p>
```

Comment:
We probably want to extend this text to mention `Zulip accounts hosted by {{EXTERNAL_HOST}}`, to address the Zulip Cloud vs. self hosted clarity question.

Reply:
@timabbott: Will `EXTERNAL_HOST` always have a value or do we need to wrap this in an `if` clause?

Reply:
Yes, it's a mandatory setting.  https://zulip.readthedocs.io/en/latest/production/install.html#step-2-install-zulip